app/main.py

```diff
@@ -1 +1,42 @@
-#  write your code here
+class Animal:
+    def __init__(self,
+                 name: str,
+                 appetite: int,
+                 is_hungry: bool = True) -> None:
+
+        self.name = name
+        self.appetite = appetite
+        self.is_hungry = is_hungry
+
+    def print_name(self) -> None:
+        print(f"Hello, I'm {self.name}")
+
+    def feed(self) -> int:
+        if self.is_hungry:
+            print(f"Eating {self.appetite} food points...")
+            self.is_hungry = False
+
+            return self.appetite
+        return 0
```

Comment:
The `feed` method correctly checks if the animal is hungry before feeding it. However, once an animal is fed, its `is_hungry` attribute is set to `False`, and the method returns the `appetite`. If the method is called again without resetting `is_hungry` to `True`, it will always return 0. Consider adding a mechanism to reset `is_hungry` after some condition or time, if applicable.